src/JuMP.jl

```diff
@@ -54,54 +54,54 @@ const _MOIVAR = MOI.VariableIndex
 const _MOICON{F,S} = MOI.ConstraintIndex{F,S}
 
 """
-    optimizer_with_attributes(optimizer_constructor, params::Pair...)
+    optimizer_with_attributes(optimizer_constructor, attrs::Pair...)
 
-Groups an optimizer constructor with the list of parameters `params`. Note that
+Groups an optimizer constructor with the list of attributes `attrs`. Note that
 it is equivalent to `MOI.OptimizerWithAttributes`.
 
 ## Examples
 
 The call `optimizer_with_attributes(Gurobi.Optimizer, "Presolve" => 0, "OutputFlag" => 0)`
-groups a Gurobi optimizer with the parameters `Presolve` with value `0` and
+groups a Gurobi optimizer with the attributes `Presolve` with value `0` and
 `OutputFlag` with value `0`. When provided to the `Model` constructor or to
 [`set_optimizer`](@ref), it creates a Gurobi optimizer and then set the
-`Presolve` parameter to value `0` and the `OutputFlag` parameter to value `0`:
+`Presolve` attribute to value `0` and the `OutputFlag` attribute to value `0`:
```

Comment:
Note that any string attributes used here are specific to the solver. One should consult the solver's documentation to find the parameters of interest.